containers/Comments/Comment.js

```diff
@@ -36,7 +36,10 @@ export const Comment = (props) => {
     {
       title: 'Report',
       icon: 'flag',
-      action: () => console.log('Report comment'),
+      action: () => {
+        props.toggleMenu();
+        props.navigation.navigate('ReportCreateMessage', {content_type: comment.content_type, object_pk: comment.object_pk});
+      },
```

Comment:
I think here you need {content_type: comment.comment_content_type, object_pk: id}, because content_type and object_pk are referring the the content object the comment belongs to, so here you are posting the report on the idea etc and not on the comment.